tests/conftest.py

```diff
@@ -193,10 +193,21 @@ def pytest_exception_interact(node, call, report):
         if device is None:
             logger.info("Could not find device in test environment, no printing logs")
         else:
+
+            def print_device_output(instance, output):
+                """Log command output of either a MenderDevice or a MenderDeviceGroup."""
+                if isinstance(instance, MenderDevice):
+                    logger.info(output)
```

Comment:
Q:
Why  don't we need a header for the client?

Reply:
There are headers in the places where this function is called. Having a header below is just to tell multiple devices apart.

Reply:
Thanks